README.md

```diff
@@ -46,6 +46,38 @@ To perform a manual setup instead follow these steps:
 * activate the virtualenv by running the `poetry shell` command
 * copy the file `.env.sample` to `.env` and value the environment variables, especially `OPENAI_API_KEY` with the secret key of OpenAI and `PGVECTOR_*` see the [Database](#database) section
 
+
+## Custom Model
+
+In addition to using the default Openai models, you have the option to integrate other model models, such as the Cohere model or any other model of your choice. Follow the steps below to set up and use a custom model in your Brevia project.
```

Comment:
Maybe
`...you have the option to integrate other Large Languag Models, such as...`